tests/snmp/test_snmp.py

```diff
@@ -0,0 +1,5 @@
+from common.utilities import wait_until
```

Comment:
can you change the file name to snmp_util.py, in pytest it treat any file begin with test_ as a test. since this file is not a test, it should not have a prefix test.